### DISCUSSION TITLE
Feature Request

### DISCUSSION THREAD

**Discussion by syedshazli** [community/community#124135] [Mobile] [answered]
Hi, I love the GitHub mobile app but can you add a feature where we can see our progress as viewed on desktop (with the green dots signifying activity) I like to see my progress and consistency and it motivates me to improve and stay active. 
<details>
<summary>ℹ️ Additional Debug Info</summary>
GitHub for iOS, Version 1.161.0 (130088629)
Device: iPhone 12 (iOS 17.4)
</details>

**Comment by mattruggio** [comment, marked as answer]
Hey @syedshazli, thanks for the feature request! I just checked and we already have this in our backlog (seems to be a frequently requested one.) I am going to add this as another +1 internally as a gentle nudge for engineering to hopefully revisit when prioritizing. Stay tuned!
